examples/gradle-v2/kotlin-multiplatform-example/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    kotlin("multiplatform") version "1.9.25"
+    id("org.jetbrains.dokka") version "2.0.20-SNAPSHOT"
+}
+
+group = "org.dokka.example"
+version = "1.0-SNAPSHOT"
+
+kotlin {
+    jvm()
+    linuxX64()
+    macosX64()
+    js(IR) {
+        browser()
+    }
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.1")
+            }
+        }
+    }
+}
+
+dokka {
+    // Create a custom source set not known to the Kotlin Gradle Plugin
+    dokkaSourceSets.register("customSourceSet") {
```

Comment:
what is the reason for this?
I don't think we need to show it in `KMP` example.
If you think it's needed, than it's better to create a separate example for custom source set creation (later)

Reply:
I have no idea, it's in the original DGPv1 example

https://github.com/Kotlin/dokka/blob/7ccc48eda17ecd82e2974002faf7e5da188ae2c5/examples/gradle/dokka-multiplatform-example/build.gradle.kts#L39-L44

I can just remove it? I'm happy either way.

Reply:
Let's drop it